drivers/scheduler.c

```diff
@@ -294,8 +294,8 @@ scheduler_register_event(scheduler_t *s, char mode, int fd,
 	event->id       = s->uuid++;
 	event->masked   = 0;
 
-	if (!s->uuid)
-		s->uuid++;
+	if (s->uuid < 0)
+		s->uuid = 1;
```

Comment:
Here as per my initial understanding of code flow, the idle way would be,  assign unused ones if
we reach the max limit.